docs/src/pages/guides/BringYourOwnPersistence.md

```diff
@@ -143,12 +155,23 @@ On success, set `MultiResourceResult.success` to true and set `MultiResourceResu
 
 On failure, set `MultiResourceResult.success` to false and set `MultiResourceResult.outcome` to an OperationOutcome with one or more issues which indicate the failure.
 
+#### Extended Operations
+
+The IBM FHIR Server supports [extended operations](https://www.hl7.org/fhir/operations.html). The IBM FHIR Server has some operations which use custom persistence interactions: 
+
+| Operation Name | Interfaces to implement |
+|----------------|-------------------------|
+| `$healthcheck` | `getHealth()` must be implemented to indicate if the persistence layer is in a healthy state and can access the data store |
+| `$erase`  | `erase` must be implemented so that ResourceEraseRecord is returned for each Erase operation.  If not implemented it throws an exception|
+| `$export` | `fetchResourcePayloads` must be implemented so that the ResourcePayload is returned as a BinaryStream |
+| `$reindex`| `isReindexSupported` must be true, and if true, it `reindex` must be implemented so that a Resource is updated after ingestion|
```

Comment:
```suggestion
| `$reindex`| `isReindexSupported` must be true and, if true, `reindex` must be implemented so that users can apply the latest search configuration to a resource after it is already ingested|
```